### DISCUSSION TITLE
Why docor.py gives errors

### DISCUSSION THREAD

**Discussion by steparkcmb** [biostars/biostar-handbook-issues#208] [Q&A]
**(bioinfo)
cmb@DESKTOP-3OOI0V0 ~
$ doctor.py
# Doctor! Doctor! Give me the news.
# Checking symptoms ...
# bwa ... OK
# datamash ... OK
# fastqc -h ... OK
# hisat2 ... OK
# seqret -h ... OK
# featureCounts ... OK
# efetch ... OK
# esearch ... OK
# samtools ... OK
# fastq-dump ... OK
# bowtie2 ... OK
# **bcftools ... ERROR**
# seqtk ... OK
# seqkit ... OK
# bio ... OK
# fastq-dump -X 1 -Z SRR1553591 ... OK
#
# Your system shows 1 error!
#
# Run the command that has failed for more details.**

You have already installed bcftools. But the error keeps popping up. What is the solution?

**Comment by ialbert** [comment]
that's a new one for sure, you can move on frankly, bcftools is used only much latter
